### ISSUE TITLE
Authorization doesn't work for me, why?

### ISSUE THREAD

**Issue by alexost-de** [IDCI-Consulting/IDCIKeycloakSecurityBundle#23]
Hello, I tried to install the bundle according to your [instructions](https://github.com/IDCI-Consulting/IDCIKeycloakSecurityBundle/blob/master/README.md) and to configure the Keycloak accordingly [keycloak-help-guide](https://github.com/IDCI-Consulting/IDCIKeycloakSecurityBundle/blob/master/Resources/docs/keycloak-help-guide.md).

When I open my protected route (e.g http://localhost/admin/edit) I am redirected to the keycloak login form as expected.

After the successful login, I am redirected back to the route "app_home", defined in the file _idci_keycloak_security.yaml_  (default_target_path)

After that I still see "Authenticated NO" in the symfony profiler panel. When I try to access the route again (http://localhost/admin/edit) I am immediately forwarded to "app_home".

**Local environment with Docker**
- Nginx: nginx:stable-alpine
- PHP: FROM php:7.4-fpm
- Symfony 4.4.29
- IDCI Keycloak Security Bundle: v1.1.0
- Keycloak: FROM jboss/keycloak:15.0.0

What is wrong?
Please help!

**Comment by alexost-de** [comment]
I also had to adjust the IDCI\Bundle\KeycloakSecurityBundle\Provider\Keycloak as following:
`    protected function getDefaultScopes(): array
    {
        return ['profile', 'email'];
    }

    protected function getScopeSeparator(): string
    {
        return ' ';
    }
`
to to eliminate the bug "invalid scope +name,email".

**Comment by maxdgt** [comment]
Hello @alexost-de 

I saw that you were able to successfully implement the package. 
I also made the modifications you recommended above.
It successfully redirects me to the keycloak login page but after I log in, but google chrome always returns the following image.

Do i need to modify something more apart from the tutorial?

![Screen Shot 2022-03-24 at 12 55 50 PM](https://user-images.githubusercontent.com/41104732/159979973-ed4fcd9c-2276-49b7-ac77-e8cc91264130.png)

Thanks for help in advance.

**Comment by maxdgt** [comment]
![Screen Shot 2022-03-24 at 12 58 18 PM](https://user-images.githubusercontent.com/41104732/159980396-e52dc9b7-5086-4f1b-aded-166aa40d5dd3.png)

**Comment by maxdgt** [comment]
Hello @alexost-de 

I saw that you were able to successfully implement the package.
I also made the modifications you recommended above.
It successfully redirects me to the keycloak login page but after I log in, but google chrome always returns the following image.

![Screen Shot 2022-03-24 at 12 59 12 PM](https://user-images.githubusercontent.com/41104732/159980532-c874273d-c1cd-4e2a-9c10-6617c3acd927.png)


Do i need to modify something more apart from the tutorial?

Thanks for help in advance.
